app/validators/uuidValidator.js

```diff
@@ -7,6 +7,4 @@ async function isUUIDValid(id) {
     }
 }
 
-const id = [param('id').custom(id => isUUIDValid(id))];
-
-module.exports = { id };
+module.exports = [param('id').custom(id => isUUIDValid(id))];
```

Comment:
Maybe it's better create more abstract validator:

module.export = (arg) => {
    return [
         param(arg).custom(id => isUUIDValid(id))
    ]
}

and then in some advanced router:

/users/:id/contracts/:contractId, [uuidValidator('id'), uuidValidator('contractId'), validate]

Reply:
Then change also text in promise.reject to 'must be valid UUID'